ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/StreamingOutput.java

```diff
@@ -21,4 +21,7 @@
 public interface StreamingOutput {
 
   void write(OutputStream output) throws IOException;
+
+  void close();
```

Comment:
nit, we should instead have this `implement AutoCloseable`

Reply:
Are you suggesting the classes implement AutoCloseable, or that the interface extends AutoCloseable? The latter makes sense to but though I got a compiler warning:
```
Warning:(21, 8) java: auto-closeable resource io.confluent.ksql.api.server.StreamingOutput has a member method close() that could throw InterruptedException
```
I worked around it with the same workaround we use in ServiceContext.java. LMK if this isn't what you had in mind.

Reply:
eh I was suggesting the latter but I'm not entirely sure I understand the warning.

Reply:
I don't fully understand it either but it seems general consensus is that the warning is bogus and the workaround we've used is fine:
https://bugs.openjdk.java.net/browse/JDK-8155591
https://bugs.openjdk.java.net/browse/JDK-7027157
https://issues.apache.org/jira/browse/CXF-7465
https://github.com/joyent/java-manta/issues/322

Going to leave this as is.